### ISSUE TITLE
Fix `at-rule-descriptor-no-unknown` false positive for `position-area` descriptor in `@position-try`

### ISSUE THREAD

**Issue by bdurrer** [stylelint/stylelint#9332]
### What minimal example or steps are needed to reproduce the bug?

```css
@position-try --custom-left {
  position-area: left;
  width: 100px;
  margin-right: 10px;
}
```

### What minimal configuration is needed to reproduce the bug?

```json
{
  "extends": [
    "stylelint-config-standard"
  ],
  "plugins": [  ],
  "rules": { }
}
```

### How did you run Stylelint?

yarn run stylelint "**/*.css"

### Which Stylelint-related dependencies are you using?

```json
{
  "name": "teststylelint",
  "dependencies": {
    "stylelint": "17.12.0",
    "stylelint-config-standard": "40.0.0"
  }
}
```

### What did you expect to happen?

There shouldn't be any error.  The example is taken directly from [Mozilla MDM](https://developer.mozilla.org/en-US/docs/Web/CSS/Reference/At-rules/@position-try#position-area)
[CSSWG](https://drafts.csswg.org/css-anchor-position-1/#at-ruledef-position-try) also mentions `position-area` being a valid property

### What actually happened?

```
src/styles/components/popover.pcss
   95:3  ✖  Unknown descriptor "position-area" for at-rule "@position-try"  at-rule-descriptor-no-unknown
```

**Comment by romainmenke** [comment]
Hi @bdurrer,

Thank you for reporting this.

I've added syntaxes for these descriptors in [css-syntax-patches-for-csstree](https://github.com/csstools/postcss-plugins/blob/main/packages/css-syntax-patches-for-csstree/CHANGELOG.md#115)

I've verified that this issue should now be fixed in the [Demo](https://stylelint.io/demo/#N4Igxg9gJgpiBcIACAHCBnAlgF0xAdgLTYBOAngASGFgCu62EAtoQDYwBm2FwAOvhQposuAoQCGJGOPgV2XANz9BAd0xRsAC1kBGAAx6UADyUCKTSQHNMREpkubsuwyf4BfEABoQHTOwBy4kxwiDBGQSjsAHRg6Ohe4AS+lgggYWgk3LAc4rSs3HxmYdgw+FDosgDaAOQMZOysNtg0SfaEDOJlklDVALqe7qYJkPjJAGIQJBbYqXUNTTGtllFMAFbx3rAo8YiFgrwgczCN+NgHsges4iUMBwNmB0cnzSPJ7didUN3nFJfXMLcQO5+CA3EA)

Updating your indirect dependencies (e.g. with `npm update`) should resolve it on your end.

Please let us now if you encounter further issues 🙇
